src/Metrics.ts

```diff
@@ -224,7 +225,7 @@ declare var Metrics: MetricsConstructor
  * ```
  */
 class BrowserMetrics implements Metrics {
-  private metrics: { [name: string]: number | string[] | undefined }
+  private metrics: { [name: string]: number | string[] | string | undefined }
```

Comment:
does it make sense to make inpel same as clsel (string[]) just for uniformity sake?